src/dashboard/Elsa.Dashboard.Web/Elsa.Dashboard.Web.csproj

```diff
@@ -8,6 +8,7 @@
         <ProjectReference Include="..\..\core\Elsa\Elsa.csproj" />
         <ProjectReference Include="..\..\dashboard\Elsa.Dashboard\Elsa.Dashboard.csproj" />
         <ProjectReference Include="..\..\persistence\Elsa.Persistence.MongoDb\Elsa.Persistence.MongoDb.csproj" />
+        <ProjectReference Include="..\..\persistence\Elsa.Persistence.YesSql\Elsa.Persistence.YesSql.csproj" />
```

Comment:
I understand you added this project reference for testing purposes, but I actually want the dashboard app to use MongoDB by default for Elsa 1. Can you please remove this reference (and any other references to YesSQL in this project)?